servlet/spring-boot/java/oauth2/azure-active-directory-b2c/web-application-1-simple-sample/src/main/java/com/spring/sample/b2c/controller/SampleController.java

```diff
@@ -0,0 +1,51 @@
+package com.spring.sample.b2c.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.annotation.RegisteredOAuth2AuthorizedClient;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import static org.springframework.security.oauth2.client.web.reactive.function.client.ServletOAuth2AuthorizedClientExchangeFilterFunction.oauth2AuthorizedClient;
+
+@Controller
+public class SampleController {
+
+    @Autowired
+    private WebClient webClient;
+
+    @GetMapping(value = { "/", "/home" })
+    public String index() {
+        return "home";
+    }
+
+    @GetMapping(value = { "/getResourceServer" })
```

Comment:
`/getResourceServer` -> `/resourceServer`

Not use verb in url.

Refs: https://cloud.google.com/blog/products/api-management/restful-api-design-nouns-are-good-verbs-are-bad